lib/javascripts/i18n.js

```diff
@@ -20,7 +21,7 @@ module.exports = Object.freeze({
     if (keyType !== 'string') {
       throw new Error(`Translation key must be a string, got: ${keyType}`);
     }
-    var template = _.at(translations, key)[0];
+    var template = at(translations, key)[0];
```

Comment:
`at` and `set` can be replaced by just simple indexing (ie. translations[key]) now since it's guaranteed to be a flattened object.